include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
I guess the question is why you would "actually want StandardC" instead of wanting "the standard definitions for the CppUTest library"...

Depends what you mean by "leads to problems" I guess.  It can't lead to actual compilation problems, by definition.  It has already led to aesthetic problems and arguments (such as this one). :grinning: 

But, since this has merged already, this ship has sailed.  So I guess the real question is whether you're willing to entertain a PR that "fixes" it or not?